build/calc_release_version.py

```diff
@@ -100,26 +102,55 @@ def check_head_tag():
 
     return None
 
+def get_next_minor (prerelease_marker):
+    """
+    get_next_minor does the following:
+    Inspect the branches that fit the convention for a release branch.
+    Choose the latest increment the minor version. Append .0 to form the new version (e.g., r1.21 becomes 1.22.0)
+    Append a pre-release marker. (e.g. 1.22.0 becomes 1.22.0-20220201+gitf6e6a7025d)
+    """
+    version_loose = LooseVersion('0.0.0')
+
+    version_new = {}
+    # Use refs (not branches) to get local branches plus remote branches
+    refs = check_output(['git', 'show-ref']).splitlines()
+    for ref in refs:
+        release_branch_match = RELEASE_BRANCH_RE.match(ref.split()[1])
+        if release_branch_match:
+            # Construct a candidate version from this branch name
+            version_new['major'] = int(release_branch_match.group('vermaj'))
+            version_new['minor'] = int(release_branch_match.group('vermin')) + 1
+            version_new['patch'] = 0
+            version_new['prerelease'] = prerelease_marker
+            new_version_loose = LooseVersion(str(version_new['major']) + '.' +
+                                                str(version_new['minor']) + '.' +
+                                                str(version_new['patch']) + '-' +
+                                                version_new['prerelease'])
+            if new_version_loose > version_loose:
+                version_loose = new_version_loose
+                if DEBUG:
+                    print('Found new best version "' + str(version_loose) \
+                            + '" based on branch "' \
+                            + release_branch_match.group('brname') + '"')
+    return str(version_loose)
+
 def main():
     """
     The algorithm is roughly:
 
-        1. Is the current HEAD associated with a tag that looks like a release
+        - Is the --next_minor flag passed? If "yes", then return the next minor
```

Comment:
Underscore to hyphen.
```suggestion
        - Is the --next-minor flag passed? If "yes", then return the next minor
```